tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -900,17 +920,103 @@ protected void compute() {
             }
         }
 
+        protected void compute(ThreadFactory inFactory) throws InterruptedException {
```

Comment:
function is too long, needs subroutines

Reply:
There's hardly any CLDR-specific code here -- what algorithm is being implemented? Has it been implemented before? Could we re-use a standard implementation instead of doing it from scratch deep inside a class that's already too tightly coupled with numerous CLDR-specific classes?